src/hal/AngleSensor.h

```diff
@@ -3,17 +3,19 @@
 
 #include "SensorInterface.h"
 
+namespace Hal {
+
 // int is just a placeholder
 class AngleSensor : public SensorInterface<double, int> {
 public:
     explicit AngleSensor(double initialAngle);
-    double GetValue(int gpio) override;
+    [[nodiscard]] double GetValue(int gpio) override;
```

Comment:
I saw your question about this unused parameter in .cpp, but I lost where the conversation was. What you could do in .cpp file:

```cpp
double AngleSensor::GetValue(int /* unused */)
{
}
```
If I'm not mistaken this should solve this scenario 😄

Reply:
I didn't know that it was possible to do that! So cool!